webapp/tests/karma/ts/modules/messages/messages-content.component.spec.ts

```diff
@@ -245,5 +246,45 @@ describe('MessagesContentComponent', () => {
       expect(removeMessageFromSelectedConversationSpy.args[0]).to.deep.equal(['message2']);
     });
   });
+
+  describe('New message section', () => {
+    it('should display the "new message" section in a conversation with a known recipient', async () => {
+      component.selectedConversation = {
+        'id': '7aeaa4bc-ae5e-4964-8771-697209d2c6dd',
+        'messages': [
+          { 'id': '68e4ae1a-bb6c-4561-98e7-3f7cdfd00486' }
+        ],
+        'contact': {
+          'doc': { 'name': 'Groot' }
+        }
+      };
+
+      fixture.detectChanges();
+      await fixture.whenStable();
+
+      const footer = fixture.debugElement.query(By.css('#message-footer'))?.nativeElement;
+      expect(footer).to.not.be.undefined;
```

Comment:
This test is the same as the following one (should hide the "new message"... in L269) 
I think this one can check that the element has the expected text or content?

Reply:
This one checks if the element exists while the other one checks that it does not.  
I thought about checking here if the element has the expected content but I'm afraid the test would know too much about implementation details :thinking:

What's your take on this?

Reply:
Right! My eyes skipped the `.not`

Yeah, `#message-footer` can hold anything as it's a layout container... we could check the feature in specific that we want to hide: `.message-form` (same div at the end, but different meaning) with inputs / buttons inside are hidden. But I'll leave it up to you :)

Reply:
btw is it requested to display a message or warning to explain the user why the section is missing? Something like: 
"Unknown recipient please check the contact is registered in the system"

Reply:
No, I don't think we have this requirement. Maybe we should ask this question in the issue and get feedback from design.